### DISCUSSION TITLE
Need a solution that can show full url in data static or assets

### DISCUSSION THREAD

**Discussion by hoanghiep1x0** [nuxt/nuxt#19043] [Questions (Nuxt 2)]
### Is your feature request related to the problem? Please describe.

Yes. I can't use it with full link crawl DOM.

### Describe the solution you want

I need a feature that can convert most url matches to absolute for example. <img:src="URL_BASE + ~asset/img.png" />
Or maybe in app.html

<script src = "{{URL_BASE}} + /file.script.js"> </script>

### Describe the alternatives you have considered

I'm not looking for an alternative other than photos or files to a 3rd server.

**Comment by treboryx** [comment]
You can use publicRuntimeConfig in your nuxt.config file and set a variable as your base url

Then, you can reference it as this.$config.<variable>
